Having invalid (but unused) go files anywhere in the project path fails packr build

Trying to use the `packr build` command in a project that has invalid (but unused) go files fails the command.

I'm using JetBrains File Templates to create some boilerplate code in my project. Those involve creating and storing invalid go files within my project structure (inside the .idea project directory).

Now, when I execute `packr build` in the project root I get the following output:

```bash
$ packr build
Error: /Users/fletcher/go/src/project/.idea/fileTemplates/file_template_1.go:1:1: illegal character U+0023 '#'
Usage:
  packr build [flags]

Flags:
  -h, --help   help for build
```

Note: `go build` works just fine

--- a/builder/builder.go
+++ b/builder/builder.go
@@ -39,7 +39,7 @@ func (b *Builder) Run() error {
 		}
 
 		base := filepath.Base(path)
-		if base == ".git" || base == "vendor" || base == "node_modules" {
+		if base == ".git" || base == "vendor" || base == "node_modules" || base == ".idea" {
 			return filepath.SkipDir
 		}
 

